{"name": "test_login_1_error[data2]", "status": "passed", "description": "异常登录", "attachments": [{"name": "log", "source": "8f8ea17e-b56e-47d0-a42e-0938392ed48a-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "7c5baf0e-9c1b-47d7-91bb-2412f21e682a-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'username': '13760246701', 'pwd': '', 'expected': '请输入密码'}"}], "start": 1571367050941, "stop": 1571367062373, "uuid": "f80bf8a3-32a2-4114-a964-f7b59bfaa06b", "historyId": "f0c9781d941377bacc9460de93147373", "testCaseId": "09a1dfa16a6f8824208fd27bb36bdeba", "fullName": "test_login.TestLogin#test_login_1_error", "labels": [{"name": "tag", "value": "demo"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-31LFQ9S"}, {"name": "thread", "value": "5408-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}